### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by jacopocarlini**
#### List of Changes

- updated README.md 
- added .puml files

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,96 @@
...
+
+
+---  
+
+## Start Project Locally 🚀
+
+### Prerequisites
+
+- Docker
+- (Optional) Python 3
```
Comment (aomegax) [inline README.md:40]: `Python 3 (required for GPD mock service)`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,96 @@
...
+
+_NOTE: to create a **GPD mock** run in the mock directory:_ `python gpd.py`
+
+## Testing 🧪
+
+### Prerequisites
+
+- maven
+- [newman](https://www.npmjs.com/package/newman)
+- [postman-to-k6](https://github.com/apideck-libraries/postman-to-k6)
```
Comment (aomegax) [inline README.md:80]: do we use it on this project?

File: docs/IUVS_TABLE.puml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+@startuml Table
+
+title IUVs Table
+
+entity IUVS_TABLE {
+    * String partitionKeys <<paIdFiscalCode>>
```
Comment (aomegax) [inline docs/IUVS_TABLE.puml:6]: `* String partitionKey <<paIdFiscalCode>>`

File: docs/ORGANIZATIONS_CONFIG_TABLE.puml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+@startuml Table
+
+title Organizations Config Table
+
+entity ORGANIZATIONS_CONFIG_TABLE {
+  * String partitionKeys ("org")
```
Comment (aomegax) [inline docs/ORGANIZATIONS_CONFIG_TABLE.puml:6]: `* String partitionKey ("org")`

File: docs/DEBT_POSITIONS_TABLE.puml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+@startuml Table
+
+title Debtor Positions Table
+
+entity DEBT_POSITIONS_TABLE {
+  * String partitionKeys <<csv_file_name>>
```
Comment (aomegax) [inline docs/DEBT_POSITIONS_TABLE.puml:6]: `* String partitionKey <<csv_file_name>>`

**Review by aomegax — CHANGES_REQUESTED** [review]
👍

**Review by pasqualespica — APPROVED** [review]
👍

**Review by aomegax — APPROVED** [review]
👍

**Pull request merged**